doc/source/tutorials/Q-Factor.ipynb

```diff
@@ -0,0 +1,435 @@
+    }
+   ],
+   "source": [
+    "C = 1e-6  # F\n",
+    "L = 1e-9  # H\n",
+    "R = 30  # Ohm\n",
+    "\n",
+    "freq = rf.Frequency(5, 5.2, npoints=201, unit='MHz')\n",
+    "media = rf.DefinedGammaZ0(frequency=freq)\n",
+    "\n",
+    "resonator = media.line(d=np.random.rand(), unit='rad') \\\n",
+    "                **media.shunt_inductor(L) ** media.shunt_capacitor(C) \\\n",
+    "                ** media.shunt(media.resistor(R)**media.short()) ** media.open()\n",
+    "\n",
+    "resonator.plot_s_db()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b291cfb1",
+   "metadata": {},
+   "source": [
+    "Analytical formulas for this case are available and given by [2,3]:\n",
+    "$$ f_{\\mathrm{res}} = \\frac{1}{2\\pi \\sqrt{L C}} $$\n",
+    "$$ Q_L = \\omega_r C R/2 $$\n",
+    "$$ Q_0 = \\omega_r R C $$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c2b5cec7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Theoretical Resonant Frequency: 5.032921210448704 MHz\n",
+      "Theoretical Loaded Q: Q_L = 474.34164902525686\n",
+      "Theoretical Unloaded Q: Q_0 = 948.6832980505137\n"
+     ]
+    }
+   ],
+   "source": [
+    "def f_res_RLC(L, C):\n",
+    "    return 1/(2*np.pi*np.sqrt(L*C))\n",
+    "\n",
+    "def QL_RLC(R, L, C):\n",
+    "    return 2*np.pi*f_res_RLC(L, C) * R * C / 2\n",
+    "\n",
+    "def Q0_RLC(R, L, C):\n",
+    "    return 2*np.pi*f_res_RLC(L, C) * R * C\n",
+    "\n",
+    "print(f'Theoretical Resonant Frequency: {f_res_RLC(L, C)/1e6} MHz')\n",
+    "print(f'Theoretical Loaded Q: Q_L = {QL_RLC(R, L, C)}')\n",
+    "print(f'Theoretical Unloaded Q: Q_0 = {Q0_RLC(R, L, C)}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d84608c3",
+   "metadata": {},
+   "source": [
+    "First, let's create a `Qfactor` object, passing the resonator Network and the type of resonator we are dealing with:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "cde6c0c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Q = rf.Qfactor(resonator, res_type='reflection')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f246179",
+   "metadata": {},
+   "source": [
+    "Note that in case of multiple resonances, it is recommanded to also pass the estimated resonance frequency and eventually the estimated (order of magnitude) Q-factor. \n",
+    "\n",
+    "Then, we use the `fit` method to fit the loaded Q-factor $Q_L$ and resonant frequency. The returned results will be usefull to deduce the unloaded Q-factor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "226c8348",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fitted Resonant Frequency: f_L = [5.03291942] MHz\n",
+      "Fitted Loaded Q-factor: Q_L = [592.92718142]\n"
```

Comment:
Why is the fit of the loaded Q so different from the theoretical value above? ("Theoretical Loaded Q: Q_L = 474.34164902525686").
The other parameters match very well, so this seems wrong.

Reply:
I think the theoretical loaded Q formula is not correct. `R` should be the total resistance in the circuit which is parallel combination of 30 ohm in the resonator and 50 ohm port impedance.

Reply:
Calculating the equation for loaded Q with *R = 50||30 Ohm = 18.75 Ohm* gives *Q_L = 296.46*. Interestingly, if you calculate the unloaded Q with *R = 18.75 Ohms*, it is *Q0 = 592.927*, which matches to fitted loaded Q. 

I think you need to use *R=30* for unloaded Q (with only internal losses) and *R=50||30* for loaded Q (with both internal and external losses), but both times using the same equation: `R * C / sqrt(L * C)`. By the way, the *2pi* cancel out elegantly.

18.75Ohm * 1uF / sqrt(1uF * 1nH) = 592.927 (loaded Q)
30Ohm * 1uF / sqrt(1uF * 1nH) = 948.683 (unloaded Q)

Now it matches the fitted results.

Reply:
Pff, of course, shame on me. Thanks.